zero_bin/.env

```diff
@@ -0,0 +1,8 @@
+AMQP_URI=amqp://localhost:5672
+ARITHMETIC_CIRCUIT_SIZE=16..23
+BYTE_PACKING_CIRCUIT_SIZE=9..21
+CPU_CIRCUIT_SIZE=12..25
+KECCAK_CIRCUIT_SIZE=14..20
+KECCAK_SPONGE_CIRCUIT_SIZE=9..15
+LOGIC_CIRCUIT_SIZE=12..18
+MEMORY_CIRCUIT_SIZE=17..28
```

Comment:
I wonder if it wouldn't be better to have it at the root level of the workspace?